Protocol.md

```diff
@@ -1197,6 +1197,8 @@ messages or early data without having to modify an existing task.
 However, this message SHOULD NOT be abused to write custom protocols.
 Tasks SHOULD support this message to be sent at any time.
 
+Every task MUST support this message type, even after handover.
```

Comment:
I would propose merging this with the previous sentence:
> Tasks MUST support this message type and SHOULD support a message of this type to be sent or received at any time.

Reply:
Done